app/models/concerns/big_query/emits_entity_events.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module BigQuery
+  module EmitsEntityEvents
+    extend ActiveSupport::Concern
+
+    included do
+      attr_accessor :event_tags
+
+      after_create do
```

Comment:
if the feature flag is off is it worth even doing all this transformation? Should we return earlier?